### PR TITLE
Update cpp-strings.json: Fixes #1983

### LINKED ISSUE #1983: C++ Strings Cheat Sheet:
@gohar94 

Can this be triggered with `string` as well as `strings`?  The name of the class is `string` so I would assume someone would look for `cpp string help`, which doesn't display this cheat sheet.

------
IA Page:  http://duck.co/ia/view/cpp_strings_cheat_sheet

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Update cpp-strings.json 
added aliases 
#1983 

-----
IA Page: https://duck.co/ia/view/cpp_strings_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@gautamkrishnar Can you include a link to the IA page in the PR description?

**Comment by gautamkrishnar** [comment]
@mintsoft done :+1: :smile:

File: share/goodie/cheat_sheets/json/cpp-strings.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "sourceName": "C++ Reference",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/cpp-strings.json]: @gautamkrishnar we can change source name to "cplusplus"

**Comment by czipperz** [comment]
Thanks!  Closing my issue

File: share/goodie/cheat_sheets/json/cpp-strings.json [hunk 2]
```diff
@@ -3,10 +3,10 @@
     "name": "C++ Strings",
     "description": "List of basic String operations in C++",
     "metadata": {
-        "sourceName": "C++ Reference",
+        "sourceName": "Cplusplus",
         "sourceUrl": "http://www.cplusplus.com/reference/string/string/"
     },
-    "aliases": ["c++ strings"],
+    "aliases": ["c++ strings","c++ string","cpp strings","cpp string"],
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/cpp-strings.json]: `cpp strings` is already handled by the filename :+1:

**Comment by gautamkrishnar** [comment]
@moollaza removed cpp strings alias :+1:

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**